### ISSUE TITLE
Question about nodes labels of graph classification datasets.

### ISSUE THREAD

**Issue by jiaruHithub** [rusty1s/pytorch_geometric#2063]
## ❓ Questions & Help

Thank you for your contribution of awesome work.
I want to obtain the nodes label of graph classification.
For example, how can I get the each node's label of D&D dataset's every graph?

**Comment by ldv1** [comment]
Maybe like this:
```

import os.path as osp

import torch
from torch_geometric.datasets import TUDataset

path = osp.join("./my_favorite_graph_data_repo", 'TU')
dataset = TUDataset(path, name='DD')

print()
print(f'Dataset: {dataset}:')
print('==================')
print(f'Number of graphs: {len(dataset)}')
print(f'Number of features: {dataset.num_features}')
print(f'Number of classes: {dataset.num_classes}')

data = dataset[0]

print('==============================================================')

print(f'Number of nodes: {data.num_nodes}')
print(f'Number of edges: {data.num_edges}')
print(f'Average node degree: {data.num_edges / data.num_nodes:.2f}')
print(f'Contains isolated nodes: {data.contains_isolated_nodes()}')
print(f'Contains self-loops: {data.contains_self_loops()}')
print(f'Is undirected: {data.is_undirected()}')
print(f'Label: {data.y.item()}')
```

**Comment by jiaruHithub** [comment]
Thank you very much! best wish!
